e2e/wdio/platform/tests/wizard-generator.e2e-spec.ts

```diff
@@ -0,0 +1,545 @@
+            checkNavigationBySteps(onChangeExample);
+        });
+
+        it('should check navigation by scrolling', () => {
+            checkNavigationByScrolling(onChangeExample);
+        });
+    });
+
+    describe('dialog example', () => {
+        it('should check basic way', () => {
+            click(dialogExample + button);
+            checkBasicWay(dialog);
+        });
+
+        it('should check required fields validation', () => {
+            click(dialogExample + button);
+            checkValidationRequiredFields(dialog);
+        });
+
+        it('should check navigation by steps', () => {
+            click(dialogExample + button);
+            checkNavigationBySteps(dialog);
+        });
+
+        it('should check navigation by buttons', () => {
+            checkNavigationByButtonsInDialog(dialogExample);
+        });
+    });
+
+    describe('customizable dialog example', () => {
+        it('should check basic way', () => {
+            click(customizableDialogExample + button);
+            checkBasicWay(dialog);
+        });
+
+        it('should check required fields validation', () => {
+            click(customizableDialogExample + button);
+            checkValidationRequiredFields(dialog);
+        });
+
+        it('should check navigation by steps', () => {
+            click(customizableDialogExample + button);
+            checkNavigationBySteps(dialog);
+        });
+
+        it('should check navigation by buttons', () => {
+            checkNavigationByButtonsInDialog(customizableDialogExample);
+        });
+    });
+
+    describe('condition dialog example', () => {
+        it('should check basic way', () => {
+            click(branchingExample + button);
+            click(dialog + select);
+            const productType = getText(listItemText);
+            click(listItem);
+            click(dialog + nextStepBtn2);
+
+            expect(getElementClass(dialog + step, 0)).toContain('completed', 'first step is not completed');
+            expect(getElementClass(dialog + step, 1)).toContain('current', 'you not moved to second step');
+            setValue(dialog + input, name);
+            setValue(dialog + input, firstAdress, 1);
+            setValue(dialog + input, secondAdress, 2);
+            click(dialog + nextStepBtn2);
+            expect(getElementClass(dialog + step, 1)).toContain('completed', 'second step is not completed');
+            expect(getElementClass(dialog + step, 2)).toContain('current', 'you not moved to third step');
+            click(dialog + select);
+            const paymentMethod = getText(listItemText);
+            click(listItem);
+            click(dialog + nextStepBtn2);
+            setValue(dialog + input, cardDetails);
+            click(dialog + nextStepBtn2);
+            click(dialog + nextStepBtn2);
+
+            expect(getText(dialog + formLabel, 0)).toBe(productType, 'value is not equal entered value');
```

Comment:
might be better to use a loop to not repeat yourself over and over
for(i = 0.....) {
expect(getText(dialog + formLabel, i)).toBe(assertionArr[i]);
}